doc/pages/options.asciidoc

```diff
@@ -388,6 +388,22 @@ are exclusively available to built-in options.
     relating to a Kakoune version greater than this value will be displayed. Versions
     are written as a single number: Like `20180413` for version `2018.04.13`
 
+*diff_add_char* `str`::
+	  _default_ ▏ +
+	  Character used to denote added lines in diffs (e.g. git gutter).
```

Comment:
doesn't seem unreasonable to add.
One alternative to these options is to add parameters to the `show-diff` and `update-diff` commands

Reply:
Yes, I thought about that too. Which direction would we prefer to go regarding this kind of things? On one side, having them as options seems nice in terms of sharing, but on the other, we shouldn’t try to make kakoune an IDE. I was honestly surprised that the `git` wrapper commands existed in the first place!

Reply:
AFAIR we do not document non-core options here, those are support script specific so they should not be in `pages/options.asciidoc`